meta-ostro/conf/distro/include/stateless.inc

```diff
@@ -498,7 +510,7 @@ passwd:         files altfiles
 group:          files altfiles
 shadow:         files altfiles
 
-hosts:          files altfiles dns
+hosts:          files mymachines resolve myhostname altfiles dns
```

Comment:
On Thu, 2016-04-14 at 03:27 -0700, juimonen wrote:
> ok... have to say it was just my own reasoning for existing
> configuration (which might be totally wrong).

It changes the system behavior (host names get looked up dynamically
even when they are configured locally).

>  So tmpfiles was copying conf file which included "files mymachines
> resolve myhostname" then our conf included "files altfiles dns" as
> next item in internal linked list of glibc. So I don't really know how
> these should be combined in 1 conf file?

I would use the original "hosts" line from systemd (because that is what
Ostro OS has used in practice so far) and then just extend "files" to
"files altfiles" -> "files altfiles mymachines resolve myhostname"

The reasoning is that this does not change anything as long a /etc/hosts
is empty. It just adds the possibility to override the system's default
entries in /usr/.../hosts.

In the commit we need to explain that this fixes a bug in the original
nsswitch.conf introduced in the "stateless" branch: that one uses as
base something that wasn't even used in Ostro.